examples/models/burgers_n2n.py

```diff
@@ -0,0 +1,276 @@
+from copy import deepcopy
```

Comment:
I'm not sure whether you want me to look at the "n2n" files or not, so I am just going to ignore them for now.